scripts/airports.ts

```diff
@@ -0,0 +1,165 @@
+import fs from "fs";
+import _ from "lodash";
+import countriesTranslation from "i18n-iso-countries";
+import { z } from "zod";
+
+const germanLocale = "de";
+const CITIES_AIRPORTS_DE = "scripts/cities_airports_de.csv";
+const AIPORTS_URL_DATA_SOURCE =
+  "https://davidmegginson.github.io/ourairports-data/airports.csv";
+
+const airportsGermanCitiesContent = fs.readFileSync(CITIES_AIRPORTS_DE, {
+  encoding: "utf-8",
+});
+
+function removeDoubleQuotes(value: unknown): string {
+  if (typeof value === "string") {
+    return value.replace(/["']/g, "");
+  }
+
+  return "";
+}
+
+const AirportDataSourceSchema = z.object({
+  iata: z.string().trim().min(1).transform(removeDoubleQuotes),
+  country_code: z.string().transform(removeDoubleQuotes),
+  airport: z.string().transform(removeDoubleQuotes),
+  latitude: z.coerce.number(),
+  longitude: z.coerce.number(),
+  city: z.string().transform(removeDoubleQuotes),
+  type: z.string().transform(removeDoubleQuotes),
+  scheduled_service: z.preprocess(
+    (val) => removeDoubleQuotes(val),
+    z.enum(["yes", "no"]),
+  ),
+});
+
+type AiportDataSource = z.infer<typeof AirportDataSourceSchema>;
+
+type Aiport = Omit<AiportDataSource, "type" | "scheduled_service"> & {
+  country: string;
+};
+
+function translateAirportName(airportName: string): string {
+  // we cannot use .endWith() since the word airport is not always at the end
+  // e.g  "San Rafael Airport (Argentina)"
+  const airportIndex = airportName.toLowerCase().indexOf("airport");
+
+  if (airportIndex !== -1) {
+    return airportName.slice(0, airportIndex) + "Flughafen";
+  }
+
+  return airportName + " Flughafen";
+}
+
+function translateAirportCity(airport: AiportDataSource): string {
+  const rows = airportsGermanCitiesContent.split("\n");
+  let city = airport.city;
+
+  rows.forEach((row) => {
+    const [iata, , cityTranslation] = row.split(",");
+    if (airport.iata === iata) {
+      city = cityTranslation;
+    }
+  });
+
+  return city;
+}
+
+function filteredLargeMediumAirports(airports: AiportDataSource[]): Aiport[] {
+  return airports
+    .filter(
+      (airport) =>
+        (airport.type === "large_airport" ||
+          airport.type === "medium_airport") &&
+        airport.scheduled_service === "yes",
+    )
+    .map((airport) => {
+      return {
+        iata: airport.iata,
+        country_code: airport.country_code,
+        country:
+          countriesTranslation.getName(airport.country_code, germanLocale) ??
```

Comment:
Related to the previous comment about capitalization of constants in this file:

```suggestion
          countriesTranslation.getName(airport.country_code, GERMAN_LOCALE) ??
```